File: CodeSignal/MakeArrayConsecutive2.java
public class MakeArrayConsecutive2 {

    int solution(int[] statues) {
        int count = 0;
        Arrays.sort(statues);
        System.out.println(statues);
        for (int i = 0 ; i < statues.length - 1; i++) {
            if (statues[i+1] - statues[i] > 1) {
                count += (statues[i+1] - statues[i] - 1);
            }
        }
        return count;
    
    }
    
    
}

File: CodeSignal/MakeArrayConsecutive2.py
def solution(statues):
    count = 0
    statues.sort()
    for i in range(len(statues) - 1):
        left = i
        right = left + 1
        if statues[right] - statues[left] > 1:
            count = count + (statues[right] - statues[left] - 1)
    
    return count